lib/webdriverio.js

```diff
@@ -64,7 +64,7 @@ let WebdriverIO = function (args, modifier) {
     }
 
     let logger = new Logger(options, eventHandler)
-    let requestHandler = new RequestHandler(options, eventHandler, logger)
+    let requestHandler = new RequestHandler(Object.assign({}, options), eventHandler, logger)
```

Comment:
One more think: right now `RequestHandler` modifies passed options. There can be some side effects after you start cloning them. You should check this